### PR TITLE
Feature/ratings

### PR DESCRIPTION
**Pull request by mnk87**
as far as i know, this feature is done, there was a bug in the VideoController i discovered this morning but that has been resolved now.

### CODE REVIEW

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Please remove not needed comments. There are still debugging things commented out, there are comments which don't seem all that helpful anymore.

File: public/js/watch_video_js.js [hunk 1]
```diff
@@ -78,4 +78,71 @@ function startstop(event){
...
+        {
+            'method': "POST",
+            'headers': {
+                'X-CSRF-TOKEN': globalCsrfToken,
+                'Content-Type': 'application/json'
+            },
+            'body': JSON.stringify(formData)
+        }
+    ).then(function (result) {
+        //console.log(result);
```
Comment (mtheunissen82) [inline public/js/watch_video_js.js]: commented code opruimen

File: public/js/watch_video_js.js [hunk 2]
```diff
@@ -78,4 +78,71 @@ function startstop(event){
...
+        document.getElementById('scoreMessage').innerHTML = 'Your rating has been posted!';
+    }).catch(function (err) {
+        //console.log(err);
```
Comment (mtheunissen82) [inline public/js/watch_video_js.js]: dito

File: resources/views/video/show.blade.php [hunk 3]
```diff
@@ -73,5 +53,7 @@ function performRating(event) {
   document.getElementById('watchButton').addEventListener("click", function(event) {
   startstop(event);
   });
+
+  //document.getElementById('score').selectedIndex = config.score - 1;
```
Comment (mtheunissen82) [inline resources/views/video/show.blade.php]: commented code verwijderen

**Review by HonkingGoose — APPROVED** [review]

**Pull request merged**